src/lib/Zikula/Bundle/FormExtensionBundle/Twig/Extension/FormExtension.php

```diff
@@ -34,20 +54,23 @@ public function __construct($container = null)
     public function getFunctions()
     {
         return [
-            new \Twig_SimpleFunction('polyfill', [$this, 'polyfill']),
+            new \Twig_SimpleFunction('polyfill', [$this, 'polyfill'])
         ];
     }
 
-    public function polyfill(array $features = ['forms'])
+    /**
+     * Adds polyfill features to be included into the page.
+     *
+     * @param array $features List of desired polyfills
+     */
+    public function polyfill(array $features = ['forms', 'forms-ext'])
```

Comment:
because the PHP syntax highlighter they use does not like the short array syntax in default values (it does not like default values for arguments with a scalar typehint either btw).
This might be a contribution idea for someone wanting to improve syntax highlighting btw.